shared/actions/login/index.js

```diff
@@ -366,8 +367,23 @@ export function registerWithUserPass () : AsyncAction {
   }
 }
 
+function TEST_MOCK_SCREEN (dispatch) {
```

Comment:
this is temp so i can just build these mock screens using our existing login flow